### DISCUSSION TITLE
Clear selection

### DISCUSSION THREAD

**Discussion by adigitoleo** [ahrm/sioyek#1041] [Q&A] [answered]
Is there a way (command) to clear the current selection in sioyek? I can't seem to find any such command in the documentation. I often quickly flip between multiple files and even this doesn't seem to clear the selection rectangle, which is a minor but persistent annoyance.

**Comment by ahrm** [comment]
Yes, using escape key.

**Reply by adigitoleo** [reply]
Thanks, that works. Which command is triggered by the escape key (curious)?

**Reply by ahrm** [reply, marked as answer]
The `escape` command :) .

**Reply by adigitoleo** [reply]
Should this be able to work then? (in keys_user.config, v2.0.0):
```
reload;escape <C-l>
```
Because it doesn't seem to (tried reversed order as well).

**Reply by ahrm** [reply]
It does for for me in the development branch. Maybe it has since been fixed?

**Reply by adigitoleo** [reply]
OK nice, I'll wait for release then. Cheers
